android/gradle.properties

```diff
@@ -22,8 +22,7 @@ kotlin.code.style=official
 
 # Maven
 GROUP=com.revenuecat.purchases
-VERSION_NAME=5.0.0-beta.6
-POM_NAME=purchases-hybrid-common
+VERSION_NAME=5.0.0-rc.1POM_NAME=purchases-hybrid-common
```

Comment:
Any idea idea why this on the same line? 🤔  I'm _thinking_ I should manually split this...

cc: @tonidero @vegaro